test/httpErrorsReply.test.js

```diff
@@ -42,7 +36,53 @@ test('Should generate the correct http error', t => {
             message: 'I\'m a teapot',
             statusCode: 418
           })
+          // should be deleted after release of https://github.com/jshttp/http-errors/pull/73
+        } else if (code === '425') {
+          t.same(JSON.parse(res.payload), {
+            error: node14 ? 'Too Early' : 'Unordered Collection',
+            message: 'Unordered Collection',
+            statusCode: 425
+          })
+        } else {
+          t.same(JSON.parse(res.payload), {
+            error: statusCodes[code],
+            message: statusCodes[code],
+            statusCode: Number(code)
+          })
+        }
+      })
+    })
+  })
+  t.end()
+})
+
+test('Should generate the correct http error using getter', t => {
+  Object.keys(statusCodes).forEach(code => {
+    if (Number(code) < 400) return
+    t.test(code, t => {
+      t.plan(3)
+      const fastify = Fastify()
+      fastify.register(Sensible)
+
+      fastify.get('/', (req, reply) => {
+        reply.getHttpError(code)
+      })
+
+      fastify.inject({
+        method: 'GET',
+        url: '/'
+      }, (err, res) => {
+        t.error(err)
+        // should be deleted after release of https://github.com/jshttp/http-errors/pull/73
```

Comment:
```suggestion
        // TODO should be deleted after release of https://github.com/jshttp/http-errors/pull/73
```

Reply:
maybe it'd be better to accept TODO comment in all places where I'm using it?

Reply:
Yes, It would be handy to fix it later on with a copy-paste!

Reply:
done